src/logger/Logger.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * @flow
+ * @file Logging Class
+ * @author Box
+ */
+
+/**
+ * Generates a GUID/UUID compliant with RFC4122 version 4.
+ *
+ * @return {string} A 36 character uuid
+ */
+function uuidv4() {
+    /* eslint-disable */
+    function generateRandom(c) {
```

Comment:
can this be defined outside of `uuidv4`? It's getting redefined every time this function is called.